src/main/java/frc/robot/Controls.java

```diff
@@ -764,10 +809,16 @@ private void configureGroundArmBindings() {
       return;
     }
     s.groundArm.setDefaultCommand(
-        s.groundArm
-            .moveToPosition(GroundArm.STOWED_POSITION)
-            .andThen(Commands.idle())
-            .withName("Ground stowed position wait"));
+        Commands.deferredProxy(
+                () ->
+                    switch (scoringMode) {
+                      case CORAL -> s.groundArm
+                          .moveToPosition(GroundArm.STOWED_POSITION)
+                          .andThen(Commands.idle())
+                          .withName("Ground stowed position wait");
+                      case ALGAE -> Commands.none();
```

Comment:
Should this be default: instead of ALGAE, just so that if future commands are added, it will work as expected.